samples/sensor/dps310/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _dps310:
+
+DPS310 Temperature and Pressure Sensor
+###################################
```

Comment:
Title underline needs to be at least as long as the title, so add some more ``###``s